src/coreclr/vm/threadsuspend.cpp

```diff
@@ -1948,6 +1948,9 @@ typedef BOOL(WINAPI* PINITIALIZECONTEXT2)(PVOID Buffer, DWORD ContextFlags, PCON
 PINITIALIZECONTEXT2 pfnInitializeContext2 = NULL;
 
 #ifdef TARGET_X86
+typedef VOID(__cdecl* PRTLRESTORECONTEXT)(PCONTEXT ContextRecord, struct _EXCEPTION_RECORD* ExceptionRecord);
```

Comment:
Is this really `__cdecl` ?

Reply:
copied from from C:\Program Files (x86)\Windows Kits\10\Include\10.0.19041.0\um\winnt.h

```c
NTSYSAPI
VOID
__cdecl
RtlRestoreContext(
    _In_ PCONTEXT ContextRecord,
    _In_opt_ struct _EXCEPTION_RECORD* ExceptionRecord
    );
```

Reply:
Ok, you right. That's unusual for Windows API...